examples/with_web_app/test.tfvars

```diff
@@ -18,33 +18,29 @@ app_gateways = {
         probe_name            = "videos-health-probe"
     }]
     appgw_http_listeners = [{
-      name               = "http_listener"
+      name               = "web_app_http_listener"
       frontend_port_name = "frontend_port"
       protocol           = "Http"
     }]
     client_name           = "launch"
     environment           = "demo"
-    location              = "East Us 2"
-    location_short        = "eastus2"
+    location              = "East Us2"
+    location_short        = "eastus"
```

Comment:
This `location_short` doesn't match the `location`, and I see a few other references to `eastus` further on in this file. Should these resources be limited to a single region or is the intent that they span regions?

Reply:
Fixed it!!